projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -462,34 +462,51 @@ public function check_comment_allowed( int $comment_id ) {
 		}
 
 		// Check for Highlander Nonce.
-		if (
-			isset( $_POST['highlander_comment_nonce'] ) &&
-			wp_verify_nonce( sanitize_text_field( wp_unslash( $_POST['highlander_comment_nonce'] ) ), 'highlander_comment' )
-		) {
-			return;
-		}
-
-		// Log the error to Log2Logstash.
-		// Related to https://github.com/Automattic/wp-calypso/issues/99436
-		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			require_once WP_CONTENT_DIR . '/lib/log2logstash/log2logstash.php';
+		if ( isset( $_POST['highlander_comment_nonce'] ) ) {
+			$valid_nonce     = false;
+			$current_user_id = get_current_user_id();
+
+			if ( wp_verify_nonce( sanitize_text_field( wp_unslash( $_POST['highlander_comment_nonce'] ) ), 'highlander_comment' ) ) {
```

Comment:
Could set `sanitize_text_field( wp_unslash( $_POST['highlander_comment_nonce'] ) ), 'highlander_comment' )` to some variable and use it later if nonce invalid.